CMakeLists.txt

```
@@ -9,6 +9,7 @@ find_package(catkin REQUIRED COMPONENTS
   roslib
   actionlib
   actionlib_msgs
+	geometry_msgs
```

It seems your editor use actual tabs for tabs which is not consistent with the rest of the file. That's why there is a difference in indent when rendered by GitHub.
```suggestion
  geometry_msgs
```

---

Good catch.  I've removed all tabs from this file (there were more), and configured my Emacs to no longer insert tab characters in cmake files.